packages/webdriverio/src/commands/browser/waitUntil.ts

```diff
@@ -68,7 +69,7 @@ export default function (
     const fn = condition.bind(this)
     let timer = new Timer(interval, timeout, fn, true)
 
-    return timer.catch((e) => {
+    return (timer as any).catch((e: Error) => {
```

Comment:
it might be better to define proper types for Timer

Reply:
definitely, in a separate PR though

Reply:
Ok